src/lib/holocene/toaster.svelte

```diff
@@ -7,16 +13,30 @@
   export let pop: Toaster['pop'];
   export let toasts: Toaster['toasts'];
   export let closeButtonLabel: string;
+  export let position: Writable<ToastPosition>;
```

Comment:
Should we be using the `toaster.position` from `stores/toaster` here instead?
```suggestion
    import { toaster } from '$lib/stores/toaster';
    $: ({position} = toaster)
```

Reply:
Will do

Reply:
✅